### PR TITLE
HPCC-16620 Add //#IMPORT to control the legacy import semantics

### PR DESCRIPTION
**Pull request by ghalliday**
Signed-off-by: Gavin Halliday <gavin.halliday@lexisnexis.com>

### CODE REVIEW

**Comment by ghalliday** [comment]
@richardkchapman please review

File: ecl/hql/hqlparse.cpp [hunk 1]
```diff
@@ -1025,6 +1046,54 @@ void HqlLex::doLine(YYSTYPE & returnToken)
...
+    if (hasPrefix(command, "import", false))
+    {
+        const char * next = skipws(command + 6);
+        if (*next == '(')
+        {
+            next = skipws(next + 1);
+            const char * bra = strchr(next, ')');
+            if (bra)
+            {
+                StringAttr option(next, bra - next);
```
Comment (richardkchapman) [inline ecl/hql/hqlparse.cpp]: Looks like you allow whitespace after the ( but not before the ), which seems a little inconsistent

**Comment by richardkchapman** [comment]
@ghalliday Couple of minor comments/questions

**Comment by ghalliday** [comment]
@richardkchapman fixed

**Pull request merged**